### PR TITLE
[WA-6]: add routing

### PR DESCRIPTION
**Pull request by Anurov-Prihodko**
and del unused style

### CODE REVIEW

File: weather-app/src/styles.scss [hunk 1]
```diff
@@ -63,5 +58,5 @@ button {
 .main-container {
   margin: 0 auto;
   padding: 0 20px;
-  width: 1170px;
+  width: 1210px;
```
Comment (dmitry-chirva) [inline weather-app/src/styles.scss:61]: Do we need this change?

Comment (Anurov-Prihodko) [reply weather-app/src/styles.scss:61]: Yes, we decided to make the content width 1170 + padding 20

File: weather-app/src/app/app.component.html [hunk 2]
```diff
@@ -1,2 +1,14 @@
-<div class="main-container"></div>
+<div class="main-container">
+  <ul>
+    <li class="list"><a class="link" routerLink="/">Home</a></li>
```
[the remark this answers is not in the document]
Comment (Anurov-Prihodko) [reply weather-app/src/app/app.component.html]: deleted

File: weather-app/src/styles.scss (hunk 1 above)
Comment (dmitry-chirva) [reply weather-app/src/styles.scss:61]: You changed 1170 to 1210 and after that => 1210 + 40 = 1250
Before we had 1170  => 1170 + 40  = 1210.

Sorry i don't understand

**Pull request merged**